### ISSUE TITLE
Provide "rolling" tag for Origin

### ISSUE THREAD

**Issue by marekjelen** [openshift/origin#14340]
When installing Origin using Ansible in containerised mode, the full version has to be specified, e.g.

```
openshift_release=v1.5.0
```

it would be nice to have images tagged using a "rolling" tag, so that when the user wants to install the latest version of OpenShift Origin 1.5, the release could be specified as

```
openshift_release=v1.5
```

##### Version
N/A

##### Steps To Reproduce
N/A

##### Current Result
User has to specified full release version when deploying Origin in containerised mode using Ansible.

##### Expected Result
User can only specify `major.minor` version for `openshift_release` in Ansible inventory when using containerised mode.
 
##### Additional Information
N/A

**Comment by mfojtik** [comment]
@smarterclayton ^

**Comment by jorgemoralespou** [comment]
Adding the installer guys. cc/ @sdodson @brenton

**Comment by sdodson** [comment]
Yes please.

**Comment by marekjelen** [comment]
yes, please ... the tags should be consistent across the components, because the Ansible playbooks are not. e.g. it's required to specify the `openshift_logging_image_version` or it will pull the latest which is not compatible with 1.5 deployments /cc @sdodson

**Comment by brenton** [comment]
@smarterclayton, are you still handling the Origin releases?  Would this simply be a few changes to the scripts under /hack?  @sdodson's team could help with any needed openshift-ansible changes.

**Comment by smarterclayton** [comment]
It's mostly changes to the hack/release.sh script to multi-tag and multi-push, but also requires some thought about cluster up.

**Comment by brenton** [comment]
We'll find someone to work on this in the next few sprints.

**Comment by marekjelen** [comment]
Worth to bring this up for 3.7?

**Comment by dobbymoodge** [comment]
@marekjelen I'm still trying to wind down my biggest time-hog duties, but I'm optimistic about this. @brenton what do you think?

**Comment by marekjelen** [comment]
@dobbymoodge @smarterclayton @brenton is there still chance to get this or should it be closed as `wontfix`?

**Comment by smarterclayton** [comment]
Did you not see the email?  This has been implemented.

**Comment by marekjelen** [comment]
@smarterclayton nope, but found it, thanks so much for getting this implemented!
